lib/utils/screen-manager.js

```diff
@@ -108,10 +97,21 @@ ScreenManager.prototype.done = function () {
   this.rl.output.write('\n');
 };
 
-function breakLines(lines) {
+ScreenManager.prototype.normalizedCliWidth = function () {
+  var width = cliWidth({
+    defaultWidth: 80,
+    output: this.rl.output
+  });
+  if (process.platform === 'win32') {
+    return width - 1;
+  }
+  return width;
+};
+
+ScreenManager.prototype.breakLines = function (lines) {
```

Comment:
How about we pass the width as a parameter instead of making those class method?